pkg/image/admission/imagepolicy/api/types.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// ImagePolicyConfig is the configuration for controlling how images are used in the cluster.
+type ImagePolicyConfig struct {
+	unversioned.TypeMeta
+
+	// ExecutionRules determine whether the use of an image is allowed in an object with a pod spec.
+	// By default, these rules only apply to pods, but may be extended to other resource types.
+	ExecutionRules []ImageExecutionPolicyRule
+	// ConsumptionRules are applied when creating resources with pod specs.
+	ConsumptionRules []ImageConsumptionPolicyRule
```

Comment:
We should remove these.  I'm not planning to review them or their usage and if they merge in this state, they won't show up in an future diff where you turn them on.